mkdocs.yml

```diff
@@ -219,6 +219,7 @@ nav:
 
             - Specific:
                 - Substitutional Point Defects in Graphene:                   tutorials/materials/specific/defect-creation-point-substitution-graphene.md
+                - Island Surface Defect Formation:                            tutorials/materials/specific/defect-creation-surface-island.md
```

Comment:
Island Surface Defect Formation in TiN

`defect-creation-surface-island-titanum-nitride.md`

Reply:
Ok.  Thought about the same.